tensilelite/Tensile/Tests/common/dot2/dot2_gfx942.yaml

```diff
@@ -0,0 +1,278 @@
+      OperationType: GEMM
+      DataType: h
+      DestDataType: h
+      ComputeDataType: s
+      HighPrecisionAccumulate: True
+      TransposeA: 1
+      TransposeB: 1
+      UseBeta: True
+      Batched: True
+      UseBias: 1
+      Activation: True
+      UseScaleAlphaVec: 1
+    - # BenchmarkProblemSizeGroup - Standard
+      InitialSolutionParameters:
+      BenchmarkCommonParameters:
+        - KernelLanguage: ["Assembly"]
+      ForkParameters:
+        - ThreadTile:
+          - [1, 1]
+          - [2, 2]
+        - WorkGroup:
+          - [64, 1, 1]
+          - [1, 64, 1]
+          - [8, 8, 2]
+          - [8, 8, 8]
+          - [2, 2, 32]
+          - [2, 2, 64]
+        - InnerUnroll: [1,2,4]
+        - AssertFree0ElementMultiple: [16]
+        - AssertFree1ElementMultiple: [16]
+        - AssertSummationElementMultiple: [1]
+        - PrefetchGlobalRead: [0,1,2]
+        - PrefetchLocalRead: [0,1]
+        - DepthU: [32,64,128,256,512]
+        - VectorWidthA: [1]
+        - VectorWidthB: [1]
+        - GlobalReadVectorWidthA: [-1,1,2,4,8]
+        - GlobalReadVectorWidthB: [-1,1,2,4,8]
+        - LocalReadVectorWidth: [2]
+        - StoreVectorWidth: [1,2]
+        - TransposeLDS: [2]
+        - StaggerUMapping: [0]
+        - StaggerU: [32]
+        - StaggerUStride: [-1]
+        - GlobalSplitU: [1,6]
+        - LdsPadA: [-1]
+        - LdsPadB: [-1]
+        - WorkGroupMapping: [1]
+        - WorkGroupMappingXCC: [1]
+        - WorkGroupMappingXCCGroup: [0]
+        - GlobalSplitUAlgorithm: ["MultipleBufferSingleKernel"]
+        - ScheduleIterAlg: [1]
+        - ScheduleLocalWrite: [1]
+      BenchmarkJoinParameters:
+      BenchmarkFinalParameters:
+        - ProblemSizes:
+          - Exact: [1280, 1280, 1, 5280]
+        - BiasTypeArgs: ['h']
+        - ActivationArgs:
+          - [Enum: Relu]
+  ########################################
+  # NN - standard
+  ########################################
+  -
+    - # ProblemType
+      OperationType: GEMM
+      DataType: h
+      DestDataType: h
+      ComputeDataType: s
+      HighPrecisionAccumulate: True
+      TransposeA: 0
+      TransposeB: 0
+      UseBeta: True
+      Batched: True
+      UseBias: 1
+      Activation: True
+      UseScaleAlphaVec: 1
+    - # BenchmarkProblemSizeGroup - Standard
+      InitialSolutionParameters:
+      BenchmarkCommonParameters:
+        - KernelLanguage: ["Assembly"]
+      ForkParameters:
+        - ThreadTile:
+          - [1, 1]
+          - [2, 2]
+        - WorkGroup:
+          - [64, 1, 1]
+          - [1, 64, 1]
+          - [8, 8, 2]
+          - [8, 8, 8]
+          - [2, 2, 32]
+          - [2, 2, 64]
+        - InnerUnroll: [1,2,4]
+        - AssertFree0ElementMultiple: [16]
+        - AssertFree1ElementMultiple: [16]
+        - AssertSummationElementMultiple: [1]
+        - PrefetchGlobalRead: [0,1,2]
+        - PrefetchLocalRead: [0,1]
+        - DepthU: [32,64,128,256,512]
+        - VectorWidthA: [1]
```

Comment:
do we support  VWA or VWB = 2 for TT = 2,2

Reply:
> VWA or VWB = 2 for TT = 2,2

Tested just now, these will lead to incorrect computation result.